### PR TITLE
UI: Control of Shares and Votes (Percent of Shares, Percent of Votes, and Types of Control)

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/17975

*Description of changes:*
- Add Types of Control checkboxes
- Update the input boxes for "percent of shares" and "percent of votes" and add validation rules
- Update text content to match the design
- Additional enhancement: re-style the Radio button border color; move text content to en.json. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-60-wwv4vjfd.web.app

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 1]
```diff
@@ -222,15 +233,26 @@ const schema = z.object({
       .refine(checkSpecialCharacters, t('errors.validation.taxNumber.specialCharacter'))
       .refine(checkTaxNumberLength, t('errors.validation.taxNumber.invalidLength'))
       .refine(validateTaxNumber, t('errors.validation.taxNumber.invalidNumber'))
-  ])
+  ]),
+  percentOfShares: z.string().refine(validatePercentage, t('errors.validation.controlPercentage.invalidPercentage')),
+  percentOfVotes: z.string().refine(validatePercentage, t('errors.validation.controlPercentage.invalidPercentage'))
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:238]: Shouldnt this be a number ?

File: btr-web/btr-main-app/components/individual-person/control/Percentage.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <UFormGroup :name="name" class="flex-col py-5">
+    <UInput
+      :id="id"
+      type="text"
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/control/Percentage.vue:5]: Shouldn't this be a number ?

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
     <div>
-      <!-- todo: move this to lang file -->
       <p class="text-justify">
-        To add a person to the transparency register of this business,
-        enter their name and email address. A request will be sent to the person at
-        their email address and they will have the ability to accept or decline the addition
-        of their information to the transparency register of the business.
-        If they accept, their information will automatically be added to the transparency register.
-        If you don’t want to send an invitation, you can add the person’s transparency information manually.
+        {{ $t('texts.addIndividualPerson') }}
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:5]: Nice, thanks for moving this.

Comment (kialj876) [reply btr-web/btr-main-app/components/individual-person/AddNew.vue:5]: +1

File: btr-web/btr-main-app/components/individual-person/control/Percentage.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    </UInput>
+  </UFormGroup>
+</template>
+
+<script setup lang="ts">
+defineProps({
+  id: { type: String, required: true },
+  name: { type: String, default: 'name' },
+  placeholder: { type: String, default: '' },
+  modelValue: { type: String, default: '' }
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/control/Percentage.vue:24]: like hrvoje mentioned above I think this should be a number

File: btr-web/btr-main-app/cypress/e2e/pages/add-individual/control-of-shares-and-votes.cy.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
+describe('pages -> Add individual', () => {
+  let en: any
+
+  beforeEach(() => {
+    // load the English version of the language file
+    cy.readFile('lang/en.json').then((json) => {
+      en = json
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/pages/add-individual/control-of-shares-and-votes.cy.ts:7]: nice

File: btr-web/btr-main-app/interfaces/type-of-control.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export interface ControlTypeI {
+    registeredOwner: boolean;
```
Comment (kialj876) [inline btr-web/btr-main-app/interfaces/type-of-control.ts:2]: rmv the semi colons pls. Also can you pls update the file name to follow the same naming convention as the other interfaces 'control-type-i.ts'

File: btr-web/btr-main-app/utils/validation.ts [hunk 7]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Check if the input value is a valid percentage number from "1" to "100"
+ * Empty string is considered valid
+ * @param value the input string
+ */
+export function validatePercentage (value: string): boolean {
```
Comment (kialj876) [inline btr-web/btr-main-app/utils/validation.ts:6]: Like mentioned above I think we want to save this value as a number so update this validation based on that

**Review by hfekete — APPROVED** [review]

**Review by kialj876 — APPROVED** [review]

**Pull request merged**